nixos/lib/testing/meta.nix

```diff
@@ -34,6 +34,13 @@ in
               Sets the [`meta.broken`](https://nixos.org/manual/nixpkgs/stable/#var-meta-broken) attribute on the [{option}`test`](#test-opt-test) derivation.
             '';
           };
+          platforms = lib.mkOption {
+            type = types.listOf types.raw;
+            default = lib.platforms.linux;
```

Comment:
We also support darwin; to be added in
- https://github.com/NixOS/nixpkgs/pull/303150.